Xamarin.Essentials/WebAuthenticator/WebAuthenticator.ios.tvos.cs

```diff
@@ -68,20 +68,32 @@ void AuthSessionCallback(NSUrl cbUrl, NSError error)
                     }
 
                     was.Start();
-                    var result = await tcsResponse.Task;
-                    was?.Dispose();
-                    was = null;
-                    return result;
+                    try
+                    {
+                        var result = await tcsResponse.Task;
```

Comment:
Why not use using

Reply:
Oh yes  🤦‍♂️. That would be a lot better.

Reply:
Hmm, but a `using` wouldn't set the value to `null` again, which seems to be a good idea. So I'd stick to the current solution.

Reply:
not sure if it needs to be nulled out or not. I assume after dispose it will get gced up and then we will reassign.

Reply:
So @jamesmontemagno you'd prefer a using instead and drop the null-assignment?

Reply:
I think should be fine with using @Redth

Reply:
Changed the implementation to `using` now and force-pushed the branch to drop the previous commit.

Reply:
/cc @jamesmontemagno @Redth